lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,29 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+      extend ActiveSupport::Concern
+
+      included do
```

Comment:
We aren't wrapping the `delegate :channel_name, to: :class` in `naming.rb` in an included block, so why do we need it here?